tools/ClangTidyHash.sh

```diff
@@ -60,7 +60,7 @@ EXIT_CODE=0
 RAN_CLANG_TIDY=no
 for run_clang_tidy in run-clang-tidy run-clang-tidy.py "${RUN_CLANG_TIDY_BIN}" ;
 do
-    if command -v "${run-clang-tidy}" > /dev/null 2>&1; then
+    if command -v ${run_clang_tidy} > /dev/null 2>&1; then
```

Comment:
You need the quotes here.  Apparently `command -v` (with no arguments) returns true, although I don't see anything in the documentation to indicate that.

Reply:
Yay for documentation :P Thanks for catching this :)